README.md

```diff
@@ -93,10 +83,67 @@ $ rake ids:update
 This rake task queries the NRDB API and creates/updates identities as appropriate.
 Identities not in the database are stripped out of ABR uploads to avoid errors.
 
+## Deploy as a web server
+- Get the project
+```shell
+git clone https://github.com/Project-NISEI/cobra.git
+cd cobra
+```
+- Set up config files
+```shell
+cp config/database.example.yml config/database.yml
+cp config/secrets.example.yml config/secrets.yml
+echo "RAILS_ENV=production" > .env
+echo "COMPOSE_FILE=prod" >> .env
+echo "POSTGRES_PASSWORD=some-good-password" >> .env
+echo "SECRET_KEY_BASE=random-64-bit-hex-key" >> .env
+echo "COBRA_DOMAIN=cobr.ai" >> .env
+echo "NISEI_DOMAIN=tournaments.nisei.net" >> .env
+```
+- Deploy
+```shell
+bin/deploy
+```
+
+## Deploy with Pulumi
+
+The deploy directory contains scripts for deploying to DigitalOcean using Pulumi for infrastructure as code. Here are
```

Comment:
mega-nit: the wrapping here is a bit odd.

Reply:
Looks right for me - my IDE wraps at 120 characters. What should it be?